src/toggle.tsx

```diff
@@ -21,7 +21,7 @@ export const Toggle: React.FC<ToggleProps> = ({
   return (
     <div
       sx={{
-        ' label': {
+        label: {
```

Comment:
better but anyway I will remove it from here, I think this is 100 times better https://github.com/nkint/margherita-ui/pull/7/files#diff-b5af72208cc9f76f372b99ad102cb505R11